meinberlin/react/budgeting/BudgetingProposalList.jsx

```diff
@@ -66,21 +69,24 @@ export const BudgetingProposalList = (props) => {
       <>
         <h2 className="visually-hidden">{translations.list}</h2>
         <ul className="u-list-reset">
+
           {data.map((proposal, idx) =>
-            <BudgetingProposalListItem
+            <ListCard
               key={'budgeting-proposal-' + idx}
-              proposal={proposal}
-              locale={meta?.locale}
+              item={proposal}
               permissions={meta?.permissions}
-              tokenvoteApiUrl={props.tokenvote_api_url}
-              onVoteChange={selectedPage => fetchProposals(selectedPage)}
               currentPage={meta?.current_page}
-              votesLeft={
-                meta?.token_info
-                  ? meta?.token_info.votes_left
-                  : false
-              }
-            />)}
+            >
+              <ListCardPill
+                badges={proposal.item_badges_for_list}
+                numOfMoreBadges={proposal.additional_item_badges_for_list_count}
+                itemUrl={proposal.url}
+              />
+              <ListCardMeta
+                item={proposal}
+                locale={meta?.locale}
+              />
+            </ListCard>)}
```

Comment:
the ListCardPill and the ListCardMeta are components that are not on every ListCard so they can be added in the app list components like this one for the budgeting